packages/store/addon/-private/system/fetch-manager.ts

```diff
@@ -136,7 +136,14 @@ export default class FetchManager {
       },
       function(error) {
         if (error instanceof InvalidError) {
-          let parsedErrors = serializer.extractErrors(store, modelClass, error, snapshot.id);
+          let parsedErrors = error.errors;
```

Comment:
Should we extract these few lines into a helper function, to avoid duplication, or not worry about it since it is just a few lines?

Reply:
the duplication with the logic in the finders? That's temporary while this feature flag still exists.